fsl/.fauna-project

```diff
@@ -1,6 +1,6 @@
 schema_directory=schema
 default=dev
 
-[stack.dev]
+[environment.dev]
```

Comment:
any plans to migrate the `stack` format to `environment`? I don't think its really worthwhile to add, I just realized that my fauna-shell file will be borked when I pull down this version.

Reply:
given we had only done the beta releases I was not planning to migrate them